src/modules/esl-toggleable/core/esl-toggleable.ts

```diff
@@ -78,11 +78,24 @@ export class ESLToggleable extends ESLBaseElement {
   /** Event to dispatch when toggleable group has changed */
   @prop('esl:change:group') public GROUP_CHANGED_EVENT: string;
 
-  /** CSS class to add on the body element */
+  /**
+   * CSS class (supports {@link CSSClassUtils}) to add on the body element
+   * */
```

Comment:
add @deprecated here?

Reply:
Not required, as we should not deprecate body class in general